src/main/java/hudson/plugins/ws_cleanup/Cleanup.java

```diff
@@ -150,7 +152,7 @@ private void doDelete(List<String> cmdList) throws IOException, InterruptedExcep
             listener.error("Cleanup command '%s' failed with code %d:", Util.join(cmdList, " "), exit);
             InputStream err = deletProc.getErrorStream();
             try {
-                Util.copyStream(err, listener.getLogger());
+                IOUtils.copy(err, listener.getLogger());
```

Comment:
:-1:
NIT: For large streams (over 2GB) has to be called `IOUtils.copyLarge(is, os)`. Log can be over 2GB very easily and it isn't a rare situation in my deployment.

Reply:
`copyStream()` is deprecated in a newer version of `Util`. That's mentioned in the commit comment.

This here is about the *error* stream, normal output should use the *output* stream. Anyway, `copy()` is using `copyLarge()` internally.

Reply:
+1 then